### PR TITLE
Ncnews 4

### PR DESCRIPTION
**Pull request by VarshaJacob**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -20,14 +20,29 @@ describe('GET /api/topics task 3',() => {
...
 });
+
+
+//4
+describe.only('GET /api/articles task 4',() => {
+    test('responds with status code 200, appropriate object properties and values',() => {
+        return request(app).get('/api/articles')
+        .expect(200)
+        .then(({body}) => {
+            expect(Array.isArray(body.articles)).toBe(true);
```
Comment (RichardNeat) [inline __tests__/app.test.js]: We don't need to explicitly test for an array here as we are using a forEach afterwards which proves that is an array. A better test may be to check the length?

File: __tests__/app.test.js [hunk 2]
```diff
@@ -20,14 +20,29 @@ describe('GET /api/topics task 3',() => {
...
+                    expect.objectContaining({
+                        author: expect.any(String),
+                        title: expect.any(String),
+                        article_id: expect.any(Number),
+                        topic: expect.any(String),
+                        created_at: expect.any(String),
+                        votes: expect.any(Number),
+                        comment_count: expect.any(String)
+                    }))
+            });
```
Comment (RichardNeat) [inline __tests__/app.test.js]: We also need a test here to check that the results are sorted in descending order by date

File: models/models.js [hunk 3]
```diff
@@ -1,9 +1,18 @@
...
     });
-};
\ No newline at end of file
+};
+
+exports.getArticleInfo = () => {
+    const sqlquery = 'SELECT articles.author,articles.title,articles.article_id,articles.topic,articles.created_at,articles.votes,COUNT(comments.article_id) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id ORDER BY articles.created_at DESC;'
+    return db.query(sqlquery)
+    .then(({rows}) => {
+        // console.log(rows)
```
Comment (RichardNeat) [inline models/models.js]: Please remove :)

**Review by RichardNeat — CHANGES_REQUESTED** [review]
Just a couple of tweaks pls :)

File: models/models.js (hunk 3 above)
Comment (VarshaJacob) [reply models/models.js]: I removed it but not sure how it cropped up; cant find in mine either :(

**Pull request merged**